### PR TITLE
Remove deprecated payment network usage. Fix #53.

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

**Comment by mnoorenberghe** [comment]
The commit message should reference #533, not 53. Also, this problem affects some of the other payment demos too e.g. shipping-options.

@rsolomakhin Any chance of a merge so that conforming browsers can use the demos?

File: paymentrequest/credit-cards/demo.js [hunk 1]
```diff
@@ -8,8 +8,6 @@ function initPaymentRequest() {
       'visa', 'mir'];
   let types = ['debit', 'credit', 'prepaid'];
   let supportedInstruments = [{
-    supportedMethods: networks,
-  }, {
     supportedMethods: ['basic-card'],
```
Comment (rsolomakhin) [inline paymentrequest/credit-cards/demo.js:11]: Could you also please change `['basic-card']` array to `'basic-card'` string? Thank you.

**Review by rsolomakhin — CHANGES_REQUESTED** [review]

**Comment by HandOfGod1369** [comment]
https://www.w3.org/TR/html51/

**Comment by HandOfGod1369** [comment]
https://www.w3.org/TR/2016/CR-secure-contexts-20160915/

**Comment by PaulKinlan** [comment]
@rsolomakhin I think this one can be closed now, the other PR fixed `supportedMethods`

**Comment by rsolomakhin** [comment]
Agreed, @PaulKinlan

**Comment by rsolomakhin** [comment]
@agektmr : Since you have the permissions here, please close this pull request. It is outdated.

**Pull request closed without merging**